pages/common/wapm.md

```diff
@@ -0,0 +1,36 @@
+# wapm
+
+> The WebAssembly package manager.
+> More information: <https://wapm.io/help/reference>.
+
+- Interactively create a wapm.toml file:
+
+`wapm init`
+
+- Download all the packages listed as dependencies in wapm.toml:
+
+`wapm install`
+
+- Download a specific version of a package and add it to the list of dependencies in wapm.toml:
+
+`wapm install {{package_name}}@{{version}}`
+
+- Download a package and install it globally:
+
+`wapm install -g {{package_name}}`
+
+- Uninstall a package and remove it from the list of dependencies in wapm.toml:
+
+`wapm uninstall {{package_name}}`
+
+- Print a tree of locally-installed dependencies:
+
+`wapm list`
+
+- List top-level globally installed packages:
+
+`wapm list -g`
```

Comment:
Is there a longer version of this flag?

Reply:
Yes, there is `--global`, I just thought I'd match the page for `npm`.

Reply:
Ah, I see. I haven't checked the `npm` page, but I think it might be best to use the long form on both pages?